deploy/helm/clickhouse-operator/values.yaml

```diff
@@ -3,92 +3,67 @@ operator:
     # operator.image.repository -- image repository
     repository: altinity/clickhouse-operator
     # operator.image.tag -- image tag (chart's appVersion value will be used if not set)
-
     tag: ""
     # operator.image.pullPolicy -- image pull policy
-
     pullPolicy: IfNotPresent
   containerSecurityContext: {}
-  # operator.resources -- custom resource configuration, look `kubectl explain pod.spec.containers.resources` for details
-
+  # operator.resources -- custom resource configuration, check `kubectl explain pod.spec.containers.resources` for details
   resources: {}
   #  limits:
-
   #    cpu: 100m
-
   #    memory: 128Mi
-
   #  requests:
-
   #    cpu: 100m
-
   #    memory: 128Mi
 
   # operator.env -- additional environment variables for the clickhouse-operator container in deployment
-
-  # possible format value [{"name": "SAMPLE", "value": "text"}]
+  # possible format value `[{"name": "SAMPLE", "value": "text"}]`
   env: []
 metrics:
   enabled: true
   image:
     # metrics.image.repository -- image repository
     repository: altinity/metrics-exporter
     # metrics.image.tag -- image tag (chart's appVersion value will be used if not set)
-
     tag: ""
     # metrics.image.pullPolicy -- image pull policy
-
     pullPolicy: IfNotPresent
   containerSecurityContext: {}
   # metrics.resources -- custom resource configuration
-
   resources: {}
   #  limits:
-
   #    cpu: 100m
-
   #    memory: 128Mi
-
   #  requests:
-
   #    cpu: 100m
-
   #    memory: 128Mi
 
   # metrics.env -- additional environment variables for the deployment of metrics-exporter containers
-
-  # possible format value [{"name": "SAMPLE", "value": "text"}]
+  # possible format value `[{"name": "SAMPLE", "value": "text"}]`
   env: []
 # imagePullSecrets -- image pull secret for private images in clickhouse-operator pod
-
-#  possible value format [{"name":"your-secret-name"}]
-
-#  look `kubectl explain pod.spec.imagePullSecrets` for details
+# possible value format `[{"name":"your-secret-name"}]`,
+# check `kubectl explain pod.spec.imagePullSecrets` for details
 imagePullSecrets: []
 # podLabels -- labels to add to the clickhouse-operator pod
-
 podLabels: {}
-# podAnnotations -- annotations to add to the clickhouse-operator pod, look `kubectl explain pod.spec.annotations` for details
-
+# podAnnotations -- annotations to add to the clickhouse-operator pod, check `kubectl explain pod.spec.annotations` for details
+# @default -- check the `values.yaml` file
```

Comment:
using `@default` here because otherwise the markdown table (when rendered in the GitHub UI) becomes too wide, and the description column is hidden / overflows and needs to be scrolled to (horizontally)

Reply:
oh, miss it thanks